axelor-base/src/main/java/com/axelor/apps/base/web/UserController.java

```diff
@@ -177,4 +177,9 @@ public Long apply(@Nullable Object input) {
       TraceBackService.trace(response, e);
     }
   }
+  
+  public void setDefaultUserRecord(ActionRequest request, ActionResponse response) {
+	  AppSettings appSettings = AppSettings.get();
+	  response.setValue("language",appSettings.get("application.locale"));
```

Comment:
Please add `try { //your code } catch { TraceBackService.trace(response, e); }` in the controller to log exceptions, then apply java google code format.